src/main/java/org/cactoos/set/Sorted.java

```diff
@@ -27,15 +27,10 @@
 import java.util.Set;
 import java.util.TreeSet;
 import org.cactoos.iterable.IterableOf;
-import org.cactoos.scalar.Unchecked;
 
 /**
  * Sorted Iterable as {@link Set}.
  *
- * <p>This class should be used very carefully. You must understand that
```

Comment:
@victornoel Maybe just a few words describing this class???
Like: based on `TreeSet`

Reply:
@fanifieiev thank you, it is done, I also added a todo so that the `SortedSet` interface is also implemented.